docs/docs/contributions/releases/release-strategy.mdx

```diff
@@ -44,6 +44,17 @@ We try our best to write bug-free code, but, like everyone, we sometimes make mi
 If you encounter a bug, please gently let us know by opening a GitHub issue or messaging us on Slack. See [Community](/community/members).
 :::
 
+### Stable release managers
+
+Our weekly [release process](./release-process.mdx) is executed by a rotating "maintainer of the week" (MOTW). But because the entire stable release process (through `dev`s, to an `a`, past `rc`s, etc) can take 8 to 12 weeks, we additionally assign a release manager per stable release to drive the manual steps of a release that are not covered by the weekly process.
```

Comment:
Totally minor but:

```suggestion
Our weekly [release process](./release-process.mdx) is executed by a rotating "maintainer of the week" (MOTW). But because the entire stable release process (through `dev`s, to an `a`, past `rc`s, etc) can take 8 to 12 weeks, we additionally assign a release manager per stable release to have an overarching view and drive the manual steps of a release that are not covered by the weekly process.
```